### DISCUSSION TITLE
Is the tutorial document available?

### DISCUSSION THREAD

**Discussion by yukkysaito** [alandefreitas/matplotplusplus#47] [Q&A]
Do you have the documentation to make this example?
https://github.com/alandefreitas/matplotplusplus/tree/master/examples

matplot++ is great. Thank you.

**Comment by alandefreitas** [comment]
I'm not sure I understand your question, but if you want to compile the library and all the examples, you can use:
```bash
mkdir build
cd build
cmake ..  -DCMAKE_BUILD_TYPE=Release
cmake --build . -j 2 --config Release
```
